fcrepo-clients/src/main/java/edu/unc/lib/dl/acl/fcrepo4/AccessControlServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2017 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.acl.fcrepo4;
+
+import edu.unc.lib.dl.acl.exception.AccessRestrictionException;
+import edu.unc.lib.dl.acl.service.AccessControlService;
+import edu.unc.lib.dl.acl.util.AccessGroupSet;
+import edu.unc.lib.dl.acl.util.ObjectAccessControlsBean;
+import edu.unc.lib.dl.acl.util.Permission;
+import edu.unc.lib.dl.fedora.PID;
+
+/**
+ * Implementation of the service to evaluate and retrieve CDR access controls in
+ * a fcrepo4 repository.
+ * 
+ * @author bbpennel
+ *
+ */
+public class AccessControlServiceImpl implements AccessControlService {
+
+	private InheritedPermissionEvaluator permissionEvaluator;
+
+	private GlobalPermissionEvaluator globalPermissionEvaluator;
+
+	@Override
+	public ObjectAccessControlsBean getObjectAccessControls(PID pid) {
+		// TODO stub
```

Comment:
It is probably going to be removed, and i will go ahead and deprecate it.  It was used in f3 code, but there was only one or two places that used it for anything more than calling hasPermission.